docs/LOGGING.md

```diff
@@ -1,43 +1,80 @@
-# Logging
+# Logging of Mountpoint for Amazon S3 CSI Driver
 
-There are two types of logging you can use for troubleshooting. The first set of logs are from the CSI Driver, and the other are from Mountpoint.
+The CSI Driver consists of three main components deployed to your cluster.
+This document explains which component you should look at in different scenarios,
+and it also explains how to obtain logs for these components.
+See [Architecture of Mountpoint for Amazon S3 CSI Driver](./ARCHITECTURE.md)
+for more details on the architecture of the CSI Driver.
 
-## CSI Driver logs
+## The Controller Component (`aws-s3-csi-controller`)
 
-By default, CSI Driver logs are written to the driver pod’s `stderr` and those are captured by Kubernetes and may be retrieved with a corresponding API call:
+The controller component is responsible for spawning or assigning Mountpoint Pods to your workloads.
+If for some reason the controller fails to perform this operation,
+you might get a `Failed to find corresponding MountpointS3PodAttachment custom resource ...` error in your workload.
 
-    POD_NAME=s3-app
-    NODE_NAME=$(kubectl get pod ${POD_NAME} -o=custom-columns=NODE:.spec.nodeName --no-headers | tail -1)
-    DRIVER_POD=$(kubectl get pods -n kube-system --field-selector spec.nodeName=${NODE_NAME} -o=custom-columns=NAME:.metadata.name | grep s3-csi-node)
-    kubectl logs ${DRIVER_POD} -n kube-system --container s3-plugin
+In that case you can get the logs of the controller component by running:
 
-## Mountpoint logs
+```bash
+$ kubectl logs -n kube-system -lapp=s3-csi-controller
```

Comment:
nit: add space between `-l` and `app` for clarity. Reference: https://kubernetes.io/docs/reference/kubectl/generated/kubectl_logs/

Also applies to other examples